Overview of dyslexia fonts (#1146)
a11yproject/a11yproject.com: src/posts/2020-10-28-dyslexia-fonts.md

---
layout: post
title: Dyslexia fonts
description: A quick overview of dyslexia fonts.
category: Assistive technology
author: Hamsa Harcourt
date: 2020-10-28
eleventyNavigation:
  key: {{ category }}
  title: {{ title }}
tags:
  - assistivetechnology
---

According to Oxford Advanced Learner's Dictionary:

> Dyslexia is a slight disorder of the brain that causes difficulty in reading and spelling, for example, but does not affect intelligence

> Dyslexia affects up to 1 in 5 people. However, dyslexia like most conditions, ranges from a mild disability to an extremely disabling one. Since people with dyslexia transpose characters, fonts are an important facet to consider.

Author: I love these feedbacks. They're really helpful. 

What do you think about me changing "People with dyslexia" to "dyslexics"

> Dyslexia affects up to 1 in 5 people. However, dyslexia like most conditions, ranges from a mild disability to an extremely disabling one. Since people with dyslexia transpose characters, fonts are an important facet to consider.

Author: Again, I find myself using "letters, numbers, punctuation, and other characters" throughout the page. Do you think this will pose a challenge to our readers?

What do you think about informing our readers beforehand that characters refers to letters, numbers, punctuation etc

> Dyslexia affects up to 1 in 5 people. However, dyslexia like most conditions, ranges from a mild disability to an extremely disabling one. Since people with dyslexia transpose characters, fonts are an important facet to consider.

Reviewer: >What do you think about me changing "People with dyslexia" to "dyslexics"

I'm unsure. I think investigating what the preferred, identity-first language is for this community, as "dyslexics" could be seen as derogatory. 

>What do you think about informing our readers beforehand that characters refers to letters, numbers, punctuation etc

I think that could work. Something like "written characters (letters, numbers, punctuation, and other symbols)" could work. What I want to stress here is that it's more than just the alphabet that is affected.

## Case study

When designing fonts, it's common practice to flip and rotate a character to form other characters. Letters b, p, d and q are usually created using this method. However, this causes problems for people with dyslexia. This is because it makes the letters easy to swap. As a result, people with dyslexia prefer fonts where all the characters are unique. 

## Introducing dyslexia fonts

Dyslexia fonts are typefaces designed specially for people with dyslexia. They are designed so that each character is unique and distinguishable. Dyslexia fonts make each character unique by using thicker lines in parts of the letters. Also, the letters are slanted a bit.  Some dyslexia fonts include: Dyslexie, Open dyslexic, Gill dyslexic, Read Regular etc.

> ![an example of dyslexia fonts](/img/posts/2020-10-28-dyslexia-fonts/dyslexia-font.png)

Reviewer: I am not seeing this image added to this PR. Can you push it up?

> However, according to recent studies, it has been proven that dyslexia fonts do **not** help dyslexics **read faster and make fewer mistakes**. In fact, most dyslexics prefer Verdana and Helvetica to  other fonts , including dyslexia fonts.

Reviewer: ```suggestion
According to recent studies, it has been proven that Dyslexia fonts do **not** help people **read faster and make fewer mistakes**. In fact, most people with Dyslexia prefer Verdana and Helvetica to  other fonts.
```

> However, according to recent studies, it has been proven that dyslexia fonts do **not** help dyslexics **read faster and make fewer mistakes**. In fact, most dyslexics prefer Verdana and Helvetica to  other fonts , including dyslexia fonts.

Reviewer: This is a huge dissertation and potentially a generalization. We need to link to these findings cited in a credible source or sources.

## General font tips

The following are general font tips that improves the overall quality of a website:

> - Font size should be 12-14pt or its equivalent.

Reviewer 2: I think this is a minimal requirement, isn't it?

```suggestion
- Font size should be at least 12-14pt or its equivalent.
```

> - Font size should be at least 12-14pt or its equivalent.

Reviewer: Why?

> - Larger line spacing improves readability.

Reviewer: How?

> - Use a small number of fonts, ideally 1 or 2.

Reviewer: Why?

> - Use **bold** to add emphasis rather than *italics* or UPPERCASE.

Reviewer: Why?

> - Animated texts must be used sparingly.

Reviewer: Why?

> 

Reviewer: I'm asking you to clarify this to provide an explanation to a reader who potentially has never heard about this before. It's one thing to say it, it's another to explain _why_ an issue impacts people. This will go a long way to helping to create understanding and empathy.

## Summary

> Dyslexia is characterized as a condition where people transpose characters. Using a specific font is not a treatment for dyslexia. This is because dyslexia is an issue with language, not vision. However, sans serif fonts have proven easiest to read on screen.

Reviewer: >Dyslexia is characterized as a condition where people transpose characters.

Where do they transpose characters?

> Dyslexia is characterized as a condition where people transpose characters. Using a specific font is not a treatment for dyslexia. This is because dyslexia is an issue with language, not vision. However, sans serif fonts have proven easiest to read on screen.

Reviewer: ```suggestion
Dyslexia is characterized as a condition where people transpose characters. Using a specific font is not a treatment for Dyslexia. This is because Dyslexia is an issue with language, not vision. However, sans serif fonts have proven easiest to read on screen.
```

> Dyslexia is characterized as a condition where people transpose characters. Using a specific font is not a treatment for dyslexia. This is because dyslexia is an issue with language, not vision. However, sans serif fonts have proven easiest to read on screen.

Reviewer: >However, sans serif fonts have proven easiest to read on screen.

This was a surprise note to end the post on. Can we work this into the earlier part of the post where we discuss what makes a good Dyslexia-friendly font? Also, we'll need to link to research for this as well.

> Dyslexia is characterized as a condition where people transpose characters. Using a specific font is not a treatment for dyslexia. This is because dyslexia is an issue with language, not vision. However, sans serif fonts have proven easiest to read on screen.

Reviewer: >This is because Dyslexia is an issue with language, not vision.

Is it? We haven't discussed this before, so we'll probably want to provide more background earlier.

---
layout: post
title: Dyslexia fonts
description: A quick overview of dyslexia fonts
category: Assistive technology
author: Hamsa Harcourt
further_reading:
  - url: https://accessibility.huit.harvard.edu/design-readability
    title: Design for readability
    source: Havard University
date: 2020-10-28
eleventyNavigation:
  key: {{ category }}
  title: {{ title }}
tags:
  - assistivetechnology
---

> According to Oxford Advanced Learner's Dictionary:

Reviewer: Can we link to the definition? This will help lend authority.

<blockquote>
  <p>
    Dyslexia is a slight disorder of the brain that causes difficulty in reading and spelling, for example, but does not affect intelligence.
  </p>
</blockquote>

> Dyslexia affects about 1 in 5 people worldwide. Its symptoms range in severity from mild to extreme. Since letters, numbers, punctuation, and other characters  are often transposed by Dyslexia, font choice is an important thing to consider.

Reviewer: ```suggestion
Dyslexia affects about 1 in 5 people worldwide. Its symptoms range in severity from mild to extreme. It transposes letters, numbers, punctuation, and other characters, which affects your ability to read. This makes selecting Dyslexic-friendly fonts an important thing to consider.
```

Another heavy edit, but what I want to communicate here is A) what Dyslexia does and B) You can do something about it with your font selection.

## Background

> When designing fonts, some typographers flip and rotate one character to form other characters. Letters b, p, d and q are usually created using this method. This is because it makes it easier to make the font. However, this practice can causes problems for people with Dyslexia. This is because some letters, numbers, punctuation, and other characters become indistinguishable. Hence, making it easy to confuse one for another. As a result, people with Dyslexia prefer fonts where all the characters are unique. 

Reviewer: ```suggestion
When designing fonts, some typographers flip and rotate one character to form other characters. Letters b, p, d and q are usually created using this method. This is because it makes it easier to make the font. 

However, this practice can causes problems for people with Dyslexia. This is because some letters, numbers, punctuation, and other characters become indistinguishable. This can make it easy to confuse one character for another. As a result, people with Dyslexia prefer fonts where all the characters are unique. 
```

> ## Introducing Dyslexia fonts

Reviewer: What do you think about a title such as "The questionable utility of Dyslexic fonts?"

My thinking is that we spend a decent chunk of this point listing what they are, including an image, which may make it seem like they work. I am thinking the the followup paragraph may be skimmed over, so we'd want to set expectations upfront.

Author: This is true! It is logical and quite thoughtful

> Dyslexia fonts are typefaces designed specially for people with Dyslexia. They are designed so that each character is unique and distinguishable. Dyslexia fonts also make each character unique by using thicker lines in some parts of the letters, as well as slanting them a bit. 

Reviewer: ```suggestion
Dyslexia fonts are typefaces designed so that each character is unique and distinguishable. Dyslexia fonts also make each character unique by using thicker lines in some parts of the letters, as well as slanting them a bit. They are intended to help people with Dyslexia.
```

Updating the copy a bit to emphasize what the desired outcome is for their designers. What the truth is may be different.

Some Dyslexia fonts include: 

- [Dyslexie](https://www.dyslexiefont.com/en/home/), 
- [Open Dyslexic](https://opendyslexic.org/), 
- [Lexie Readable](http://www.k-type.com/fonts/lexie-readable/), and
- [Read Regular](http://www.readregular.com/english/intro.html).

![an example of Dyslexia fonts](../img/posts/2020-10-28-dyslexia-fonts/dyslexia-font.png)
 

> According to recent studies, it has been proven that Dyslexia fonts do **not** help people **read faster and make fewer mistakes**. In fact, most people with Dyslexia prefer Verdana and Helvetica to  other fonts. For  a  detailed  review of the case study, see  [Good Fonts for Dyslexia (opens PDF)](http://dyslexiahelp.umich.edu/sites/default/files/good_fonts_for_dyslexia_study.pdf)

Reviewer: What about Comic Sans? I know some people prefer it, as well. Here's a good post on the subject I was just reading: https://www.thecut.com/2020/08/the-reason-comic-sans-is-a-public-good.html

Author: Thank you for your observation. The resource was also helpful. Additionally, I've done a quick research about comic sans. I think I'll add it to the list.

## General font tips

> The following are general font tips that improves the overall quality of a website:

Reviewer: ```suggestion
The following are general font tips that improves the overall quality of your website's typography:
```

> - Font size should be at least 12-14pt or its equivalent. This is because people with cognitive disabilities, language and learning disabilities, and low vision  cannot perceive the text. 

Reviewer: ```suggestion
- Font size should be at least 12-14pt or its equivalent. This help people with cognitive disabilities, language and learning disabilities, as well as people with low vision who may not be able to perceive the text. 
```

> - Larger line spacing improves readability. Proper line spacing makes the[ web content](https://en.wikipedia.org/wiki/Web_content) flow smoothly and ensures that the visually rendered text is presented in such a manner that is perceived without its layout interfering with its readability.

Reviewer: How much line spacing is enough? How much is too much?

> - Use a small number of fonts, ideally 1 or 2. This is because multiple fonts makes a website look unstructured and unprofessional.

Reviewer: Are there any cognitive drawbacks to a large number of fonts?

> - Use **bold** to add emphasis rather than *italics* or UPPERCASE. This is because decorated text can be difficult to read. Also, when words are written in all caps, they have a uniform rectangular shape meaning readers can't identify words by their shape. Consequently, this retards the speed of scanning and reading of readers.

Reviewer: ```suggestion
- Use **bold** to add emphasis rather than *italics*. This is because large amounts of decorated text can be difficult to read. 
- Avoid large amounts of UPPERCASE text. When words are written in all caps, they have a uniform rectangular shape, meaning some readers may not be able to identify words by their shape.
```

Broke out the UPPERCASE portion into its own bullet, as it's different than the italics portion. 

Also, we avoid using phrases like "retards." The term has strong ableist connotations.

---
layout: post
title: Dyslexia fonts
description: A quick overview of dyslexia fonts
category: Assistive technology
author: Hamsa Harcourt
further_reading:
  - url: https://accessibility.huit.harvard.edu/design-readability
    title: Design for readability
    source: Havard University
  - url: https://eyeondesign.aiga.org/sad-but-true-comic-sans-might-just-be-the-best-font-for-dyslexics/
    title: Comic Sans Might Just be the Best Font for People with Dyslexia
    source: AIGA Eye on Design
date: 2020-10-28
eleventyNavigation:
  key: {{ category }}
  title: {{ title }}
tags:
  - assistivetechnology
---

<blockquote>
  <p>
    Dyslexia is a slight disorder of the brain that causes difficulty in reading and spelling, for example, but does not affect intelligence.
  </p>
		<footer>
		<cite><a href="https://www.oxfordlearnersdictionaries.com/definition/english/dyslexia?q=dyslexia">Dyslexia - Oxford Advanced Learner's Dictionary</a></cite>
	</footer>
</blockquote>

> Dyslexia affects about 1 in 5 people worldwide. Its symptoms range in severity from mild to extreme. It transposes letters, numbers, punctuation, and other characters, which affects your ability to read. This makes selecting Dyslexic-friendly fonts an important thing to consider.

Reviewer: Does it make Dyslexic-friendly fonts important to consider? Later on you reveal that they're not effective. It might be better to say something like, "This means that making Dyslexic-friendly fonts and typography choices important to consider.", as those are the meaningful changes a designer can do.

Reviewer: @HarcourtHamsa Tagging you on unresolved feedback.

## Background

When designing fonts, some typographers flip and rotate one character to form other characters. Letters b, p, d and q are usually created using this method. This is because it makes it easier to make the font. 

> However, this practice causes problems for people with Dyslexia. This is because some letters, numbers, punctuation, and other characters become indistinguishable. This can make it easy to confuse one character for another. As a result, people with Dyslexia prefer fonts where all the characters are unique. 

Reviewer: @HarcourtHamsa Tagging you on unresolved feedback.

## The questionable utility of Dyslexic fonts?

Dyslexia fonts are typefaces designed so that each character is unique and distinguishable. Dyslexia fonts also make each character unique by using thicker lines in some parts of the letters, as well as slanting them a bit. They are intended to help people with Dyslexia.

Some Dyslexia fonts include: 

- [Dyslexie](https://www.dyslexiefont.com/en/home/), 
- [Open Dyslexic](https://opendyslexic.org/), 
- [Lexie Readable](http://www.k-type.com/fonts/lexie-readable/), and
- [Read Regular](http://www.readregular.com/english/intro.html).

![an example of Dyslexia fonts](../img/posts/2020-10-28-dyslexia-fonts/dyslexia-font.png)
 

> According to recent studies, it has been proven that Dyslexia fonts do **not** help people **read faster and make fewer mistakes**. In fact, most people with Dyslexia prefer Comic Sans, Verdana and Helvetica to other fonts. For  a  detailed  review of the case study, see  [Good Fonts for Dyslexia (opens PDF)](http://dyslexiahelp.umich.edu/sites/default/files/good_fonts_for_dyslexia_study.pdf)

Author: Makes sense! We don't want our readers to end up focusing on the wrong stuff

> According to recent studies, it has been proven that Dyslexia fonts do **not** help people **read faster and make fewer mistakes**. In fact, most people with Dyslexia prefer Comic Sans, Verdana and Helvetica to other fonts. For  a  detailed  review of the case study, see  [Good Fonts for Dyslexia (opens PDF)](http://dyslexiahelp.umich.edu/sites/default/files/good_fonts_for_dyslexia_study.pdf)

Reviewer: @HarcourtHamsa Tagging you on unresolved feedback.

## General font tips

The following are general font tips that improves the overall quality of your website's typography:

- Font size should be at least 12-14pt or its equivalent. This help people with cognitive disabilities, language and learning disabilities, as well as people with low vision who may not be able to perceive the text.  

> - Larger line spacing improves readability. Proper line spacing makes the[ web content](https://en.wikipedia.org/wiki/Web_content) flow smoothly and ensures that the visually rendered text is presented in such a manner that is perceived without its layout interfering with its readability. According to the [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) Line height (line spacing) should be at least 1.5 times the font size.

Author: I really need to focus on my vocabulary. Lol 😂

> - Larger line spacing improves readability. Proper line spacing makes the[ web content](https://en.wikipedia.org/wiki/Web_content) flow smoothly and ensures that the visually rendered text is presented in such a manner that is perceived without its layout interfering with its readability. According to the [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) Line height (line spacing) should be at least 1.5 times the font size.

Author: Yes we can

> - Larger line spacing improves readability. Proper line spacing makes the[ web content](https://en.wikipedia.org/wiki/Web_content) flow smoothly and ensures that the visually rendered text is presented in such a manner that is perceived without its layout interfering with its readability. According to the [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) Line height (line spacing) should be at least 1.5 times the font size.

Reviewer: @HarcourtHamsa Tagging you on unresolved feedback.

- Use a small number of fonts, ideally 1 or 2. This is because multiple fonts makes a website look unstructured and unprofessional. Also, multiple fonts obstructs the flow of content of the web page as people spend time adjusting to each new font. Hence, reducing readability. 

> - Use **bold** to add emphasis rather than *italics*. This is because large amounts of decorated text can be difficult to read. 

Reviewer: @HarcourtHamsa Tagging you on unresolved feedback.

- Avoid large amounts of UPPERCASE text. When words are written in all caps, they have a uniform rectangular shape, meaning some readers may not be able to identify words by their shape.

> - Animated texts must be used sparingly. This is because animations can be distracting espcially for a person with [AD/HD](https://www.cdc.gov/ncbddd/adhd/facts.html).

Author: This was a large edit. Thank you!

> - Animated texts must be used sparingly. This is because animations can be distracting espcially for a person with [AD/HD](https://www.cdc.gov/ncbddd/adhd/facts.html).

Reviewer: @HarcourtHamsa Tagging you on unresolved feedback.

## Summary

> Dyslexia is a [Phonological Processing disorder](https://www.nicklauschildrens.org/conditions/phonological-process-disorders). This means it is an issue with language processing, not vision. Hence using a specific font is not a treatment for Dyslexia. However, most people with Dyslexia prefer Verdana and Helvetica to other fonts.

Author: Yes we can

> Dyslexia is a [Phonological Processing disorder](https://www.nicklauschildrens.org/conditions/phonological-process-disorders). This means it is an issue with language processing, not vision. Hence using a specific font is not a treatment for Dyslexia. However, most people with Dyslexia prefer Verdana and Helvetica to other fonts.

Reviewer: @HarcourtHamsa Tagging you on unresolved feedback.
